### PR TITLE
Add transaction logs view

### PR DESCRIPTION
**Pull request by csillag**
Design:

image

Current implementation:

image

There is still some distance to go, but it's already better than nothing, I think.

### CODE REVIEW

**Comment by csillag** [comment]
There are quite a few open questions, but I think this might even be worthy of merging as it is, as a starting point,
since there is lot's of valid information already available.

File: src/app/components/Transactions/LogEvent.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+                      </TableRow>
+                    </TableHead>
+                    <TableBody>
+                      {event.evm_log_params.map((param, index) => (
+                        <TableRow key={`param-${index}`}>
+                          <TableCell>{param.name}</TableCell>
+                          <TableCell>{param.evm_type}</TableCell>
+                          <TableCell>
+                            {
+                              // TODO: how do I know if it's indexed?
```
Comment (lukaw3d) [inline src/app/components/Transactions/LogEvent.tsx:68]: Discussed multiple times by now. Remove

Comment (csillag) [reply src/app/components/Transactions/LogEvent.tsx]: Done.

File: src/app/components/Transactions/LogEvent.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+import TableRow from '@mui/material/TableRow'
+import TableCell from '@mui/material/TableCell'
+import TableBody from '@mui/material/TableBody'
+import { AccountLink } from '../Account/AccountLink'
+import { CopyToClipboard } from '../CopyToClipboard'
+
+const EvmEventParamData: FC<{ layer: Layer; param: EvmEventParam }> = ({ layer, param }) => {
+  switch (param.evm_type) {
+    case 'address':
+      return <AccountLink address={param.value as string} layer={layer} />
```
Comment (lukaw3d) [inline src/app/components/Transactions/LogEvent.tsx:23]: We should probably move monospace styling into `AccountLink`

Comment (csillag) [reply src/app/components/Transactions/LogEvent.tsx:23]: Done.

File: src/app/components/Transactions/LogEvent.tsx [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    case RuntimeEventType.consensus_accountswithdraw:
+    default:
+      return <pre>{JSON.stringify(event, null, ' ')}</pre>
+  }
+}
+
+export const TransactionLogEvent: FC<{ layer: Layer; event: RuntimeEvent; first: boolean }> = ({
+  layer,
+  event,
+  first,
```
Comment (lukaw3d) [inline src/app/components/Transactions/LogEvent.tsx:102]: isFirst, or hasDivider

File: src/app/components/Transactions/LogEvent.tsx (hunk 2 above)
Comment (csillag) [reply src/app/components/Transactions/LogEvent.tsx:23]: Re: monostace vs AccountLink: see also https://github.com/oasisprotocol/explorer/pull/225

File: src/app/components/Transactions/LogEvent.tsx [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+}) => {
+  const { t } = useTranslation()
+  const theme = useTheme()
+  const isMobile = useMediaQuery(theme.breakpoints.down('sm'))
+  const decoded = true // TODO: how do I know if this has been successfully decoded?
```
Comment (lukaw3d) [inline src/app/components/Transactions/LogEvent.tsx:107]: API is currently not returning unparsed events. Does your typescript usage ensure that we will know if API starts returning new types of events?

File: src/app/components/Transactions/Logs.tsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  const { isLoading, data } = eventsQuery
+  return <TransactionLogsView layer={transaction.layer} events={data?.data?.events} isLoading={isLoading} />
+}
+
+export const TransactionLogsView: FC<{
+  layer: Layer
+  events: RuntimeEvent[] | undefined
+  isLoading: boolean
+}> = ({ layer, events, isLoading }) => {
+  // const { t } = useTranslation()
```
Comment (lukaw3d) [inline src/app/components/Transactions/Logs.tsx:28]: Leftover

**Review by lukaw3d — APPROVED** [review]

File: src/app/components/Transactions/LogEvent.tsx (hunk 3 above)
Comment (csillag) [reply src/app/components/Transactions/LogEvent.tsx:99]: OK, `isFirst` it is.

File: src/app/components/Transactions/Logs.tsx (hunk 5 above)
Comment (csillag) [reply src/app/components/Transactions/Logs.tsx:28]: OK

File: src/app/components/Transactions/LogEvent.tsx (hunk 4 above)
Comment (csillag) [reply src/app/components/Transactions/LogEvent.tsx:118]: >  Does your typescript usage ensure that we will know if API starts returning new types of events?

Now it does.

**Pull request merged**